src/Stratis.Bitcoin.Features.PoA/Voting/IdleFederationMembersKicker.cs

```diff
@@ -127,41 +127,49 @@ public bool ShouldBeKicked(PubKey pubKey, uint blockTime, out uint inactiveForSe
 
         private void OnBlockConnected(BlockConnected blockConnectedData)
         {
-            // Update last active time.
-            uint timestamp = blockConnectedData.ConnectedBlock.ChainedHeader.Header.Time;
-            PubKey key = this.slotsManager.GetFederationMemberForBlock(blockConnectedData.ConnectedBlock.ChainedHeader, this.votingManager).PubKey;
-            this.fedPubKeysByLastActiveTime.AddOrReplace(key, timestamp);
+            try
```

Comment:
There is an exception happening here, so we need to log it as the error subscription handler isnt doing a good job